lib/ui.js

```diff
@@ -36,7 +36,7 @@ p.on('ready', () => {
 onChange(elems.key, (key) => {
   let buf;
   try {
-    buf = Buffer.from(key, 'hex');
```

Comment:
I think we should ignore spaces too, then! Would you care to add this too?

Minor nit: please use single quotes.